geojson/utils.py

```diff
@@ -54,3 +54,115 @@ def map_coords(func, obj):
     else:
         raise ValueError("Invalid geometry object %s" % repr(obj))
     return {'type': obj['type'], 'coordinates': coordinates}
+
+
+def generate_random(featureType, numberFeatures=1,
+                    numberVertices=3,
+                    boundingBox=[-180.0, -90.0, 180.0, 90.0]):
+    """
+    Generates random geojson features depending on the parameters
+    passed through.
+
+    :param featureType: A geometry type
+    :type string: Point, LineString, Polygon
+    :param numberFeatures: The number of features that will be returned
+    :type int: defaults to 1
+    :param numberVertices: The number vertices that
+    a linestring or polygon will have
+    :type int: defaults to 3
+    :param boundingBox: A bounding box in which features will be restricted to
+    :type string: defaults to the world - [-180.0, -90.0, 180.0, 90.0]
```

Comment:
Not sure @frewsxcv :) 
are you referring to a particular line? 
Your comment is a bit cyptic to this novice python-er